libs/api/database/neo4j/src/lib/services/query-repository/query-repository.service.ts

```diff
@@ -10,7 +15,7 @@ class QueryRepositoryService implements OnApplicationShutdown {
   ) {}
 
   onApplicationShutdown(): void {
-    console.log('Neo4j Shutdown');
+    Logger.log(`${this.constructor.name} ❌ Shutdown`, 'Neo4jModule');
```

Comment:
Use custom logger service

Reply:
Cannot use logger service due to circular dependencies despite using `forwardRef()`